src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -118,10 +121,15 @@ internal class TaskReceiver(tasksDirectory: String,
         }
 
         val md5sum = dockerProcessor.getMd5Sum(rootFile)
-        this.callback("md5 sum of root file", md5sum)
+        val statusCode = 0
```

Comment:
What's the point of this constant which is always 0?

Reply:
I'm using this thing in two places, I didn't want to pass 0 twice.
Next time, when I'll be able to get non-zero status code it seems, that I'll need same code in logger and getResultData

Reply:
My major point is that it is useless as status code indicator if it is always zero, no matter what happened. 
Minor point is that if you need some constant for successful or unsuccessful status codes, create an enum with semantic names: OK, FAILURE, etc.

Reply:
Replaced with enum. 
Done